third_party/closure-compiler/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,36 @@
+import java.util.HashSet;
+import java.util.List;
+
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.common.collect.ImmutableList;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+
+  private static final List<String> LOG_STRIP_TYPES = ImmutableList.of(
+      "dev.fine",
+      "dev.assert");
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
```

Comment:
@dvoytenko absolutely. trying to get a local green test first before since local is super flakey right now =/. i also need to add its integration with the gulp build pipeline